packages/react/src/components/UIShell/components/styles/_side-nav.scss

```diff
@@ -21,3 +28,46 @@ $prefix: 'cds' !default;
 .#{$prefix}--side-nav--collapsible.#{$prefix}--side-nav--expanded {
   inline-size: convert.to-rem(256px);
 }
+
+//----------------------------------------------------------------------------
+// Side-nav Panel
+//----------------------------------------------------------------------------
+.#{$prefix}--side-nav--panel {
+  .#{$prefix}--side-nav__icon:not(.#{$prefix}--side-nav__submenu-chevron) {
+    margin-inline-end: mini-units(2);
+  }
+
+  .#{$prefix}--side-nav__item.#{$prefix}--side-nav__item--icon
+    a.#{$prefix}--side-nav__link {
+    padding-inline-start: mini-units(8);
```

Comment:
```suggestion
    padding-inline-start: $spacing-10;
```

Reply:
Check this value also, I think it might be too large? Maybe spacing-05 / 1rem is what we want looking at the spec?
<img width="308" alt="Screenshot 2025-02-04 at 11 54 45 AM" src="https://github.com/user-attachments/assets/f2abdf9e-74a5-44f7-8957-3796c698df55" />

Reply:
I think the padding is correct?

![alt text](image.png)

Reply:
Oh wait, you're right, start is correct, its the end padding that might need adjusting.

Reply:
I haven't been able to replicate the end spacing in the screenshot

<img width="285" alt="image" src="https://github.com/user-attachments/assets/93d4be68-4b30-401a-9c6b-9f0e4ccb9457" />

Reply:
Weird, I'm not seeing it anymore either.